config/esp32/components/chip/CMakeLists.txt

```diff
@@ -101,6 +101,10 @@ if(CONFIG_DISABLE_IPV4)
     chip_gn_arg_append("chip_inet_config_enable_ipv4"        "false")
 endif()
 
+if(CHIP_CODEGEN_PREGEN_DIR)
+    chip_gn_arg_append("chip_code_pre_gegenerated_directory"  "\"${CHIP_CODEGEN_PREGEN_DIR}\"")
```

Comment:
:%s/chip_code_pre_gegenerated_directory/chip_code_pre_generated_directory

^ You want to do this across this PR